site/en/docs/devtools/recorder/index.md

```diff
@@ -160,6 +168,51 @@ There are options to add and remove steps too. This is useful if you want to add
    {% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/zeK91O21jvc5OD9HuovR.png", alt="The new step for coffee checkout has been updated with the aforementioned details.", width="800", height="663" %}
 6. [Replay](#replay) the flow now to see the changes.
 
+## Share user flows {: #share-flows }
+
+You can export and import user flows. This is useful for bug reporting because you can share an exact record of the steps that reproduce a bug.
+
+### Export a user flow {: #export-flows }
+
+To export a user flow:
+
+1. Open the user flow you want to export.
+2. Click on the **Export**<span class="material-icons">file_download</span> button at the top of the **Recorder** panel.
+    {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/LJJqpZ0vOEdp6rdDLXmP.png", alt="Download format options", width="800", height="490" %}
+ 1. Select one of the following formats from the dropdown list:
+    - **Export as a JSON file**. Download the recording as a JSON file.
+    - **Export as a @puppeteer/replay script**. Download the recording as a [Puppeteer Replay](https://github.com/puppeteer/replay) script.
+    - **Export as a Puppeteer script**. Download the recording as a [Puppeteer](https://pptr.dev/) script.
+1. Save the file.
+
+{% Aside 'gotchas' %}
+The [@puppeteer/replay](https://github.com/puppeteer/replay) is a library built on top of [Puppeteer](https://pptr.dev/). It is also a command line tool, so you can replay JSON files with it too.
+
+**Advance use case: Integrate with the Puppeteer Replay library**
+
+You can build your own library on top of the Puppeteer Replay library to replay or "stringify" the user flow JSON files (i.e. convert them to something else).
+
+For example, [@cypress/chrome-recorder](https://github.com/cypress-io/cypress-chrome-recorder) is a library built on top of Puppeteer Replay. You can use it to convert user flow JSON files to Cypress test scripts.
+
+Learn more about the [stringify feature in Puppeteer Replay](https://github.com/puppeteer/replay#stringify-a-recording-as-a-puppeteer-script).
+
+{% endAside %}
+
+You can do the following with each export option:
+
+- **JSON**. Edit the human-readable JSON object and [import](#import-flows) the JSON file back to the **Recorder**.
+- **@puppeteer/replay**. Replay the script with the [Puppeteer Replay](https://github.com/puppeteer/replay) library. When exporting as a @puppeteer/replay script, the steps remain a JSON object. This option is perfect if you want to integrate with your CI/CD pipeline but still have the flexibility to edit the steps as JSON and import them back into the **Recorder**.
+- **Puppeteer** script. Replay the script with [Puppeteer](https://pptr.dev/). Since the steps are converted into JavaScript, you can have more fine-grained customization, for example, export to PDF or integrate with Lighthouse. One caveat, you can't import this script back into the **Recorder**.
```

Comment:
```suggestion
- **Puppeteer** script. Replay the script with [Puppeteer](https://pptr.dev/). Since the steps are converted into JavaScript, you can have more fine-grained customization, for example, looping the steps**.
```